instances/devhub.near/widget/config/css.jsx

```diff
@@ -0,0 +1,28 @@
+const Theme = styled.div`
+  --theme-color: rgb(4, 164, 110);
+
+  .theme-btn {
```

Comment:
Also just noticing that there could be a single "CSS" module defined in devhub.near, then Theme just needs to define the color variables